lib/page/static.js

```diff
@@ -4,6 +4,10 @@ var serveStatic = require('serve-static');
 
 module.exports = function(link, page, options){
   var router = page.router;
-  router.use(serveStatic( path.join(link.path,'www','public')));
+  router.use(serveStatic( path.join(link.path,'www','public'), {
+    setHeaders: function (res, path) {
+      res.header('Access-Control-Allow-Origin', '*');
```

Comment:
i will have to divide it into 3 different group, because we have emails assets, alexa cards, resources for the web page